pkg/expression/builtin_info.go

```diff
@@ -187,14 +189,22 @@ func (b *builtinCurrentUserSig) Clone() builtinFunc {
 	return newSig
 }
 
+// RequiredOptionalEvalProps implements the RequireOptionalEvalProps interface.
+func (b *builtinCurrentUserSig) RequiredOptionalEvalProps() (set OptionalEvalPropKeySet) {
```

Comment:
What if the coder forgot to implement this interface, what will happen?

Reply:
```
type builtinCurrentRoleSig struct {
	baseBuiltinFunc
	contextopt.CurrentUserPropReader
}
```

`CurrentUserPropReader` also has a method `RequiredOptionalEvalProps`. If code does not write this method manually, the build will fail.

Reply:
What if the coder does not define `XXPropReader`?

Reply:
It may have a problem... But I think we can remove the default `RequiredOptionalEvalProps` or make the `RequiredOptionalEvalProps` panic. But it will give a lot of diff... I think it's better to do it in a new PR